doc/man/hirte-agent.conf.5.md

```diff
@@ -10,29 +10,21 @@ The basic file definition used to bootstrap hirte-agent.
 
 ## Format
 
-The hirte-agent configuration file is using the .ini file format.
+The hirte-agent configuration file is using the shell configuration file format.
```

Comment:
Is this change still correct?

Reply:
Changed to systemd configuration files format: https://www.freedesktop.org/software/systemd/man/systemd.syntax.html